tests/field_deconstruction/tests.py

```diff
@@ -28,6 +28,27 @@ def test_name(self):
         name, path, args, kwargs = field.deconstruct()
         self.assertEqual(name, "author")
 
+    def test_db_tablespace(self):
+        field = models.Field()
+        _, _, args, kwargs = field.deconstruct()
+        self.assertEqual(args, [])
+        self.assertEqual(kwargs, {})
+        # With a defined DEFAULT_DB_TABLESPACE.
+        with self.settings(DEFAULT_DB_TABLESPACE='foo'):
+            _, _, args, kwargs = field.deconstruct()
+            self.assertEqual(args, [])
+            self.assertEqual(kwargs, {})
+        # With a db_tablespace.
+        field = models.Field(db_tablespace='foo')
+        _, _, args, kwargs = field.deconstruct()
+        self.assertEqual(args, [])
+        self.assertEqual(kwargs, {'db_tablespace': 'foo'})
+        # With a db_tablespace equals to DEFAULT_DB_TABLESPACE.
+        with self.settings(DEFAULT_DB_TABLESPACE='foo'):
+            _, _, args, kwargs = field.deconstruct()
+            self.assertEqual(args, [])
+            self.assertEqual(kwargs, {'db_tablespace': 'foo'})
+
```

Comment:
I would put the test in a separate commit so it's clear it's unrelated to the code changes.

There's also some comments in tests like:
```
# We can't test the DEFAULT_TABLESPACE and DEFAULT_INDEX_TABLESPACE settings
# because they're evaluated when the model class is defined. As a consequence,
# @override_settings doesn't work, and the tests depend"
```
Does this change change that at least in part?